### DISCUSSION TITLE
Changing the URL from localhost

### DISCUSSION THREAD

**Discussion by metzhadar** [giongto35/cloud-game#281] [Q&A] [answered]
How can I change the default url from localhost ("ping server address: http://localhost:9000/echo")? I'd like to put the workers behind ingress.

**Comment by sergystepanov** [comment, marked as answer]
You could try to hack some params in the config file (configs/config.yaml). I think here is this param:

https://github.com/giongto35/cloud-game/blob/3f1d6b0205e6740a7773ea950d2774113d9636b4/configs/config.yaml#L36-L37

And here is some additional info that could be useful https://github.com/giongto35/cloud-game/issues/260#issuecomment-757476210.
Also, if you are trying to run it on a k8s cluster, we haven't tested it specifically for that but we have it in the future plans. So it may or may not work without some code changes.

**Reply by metzhadar** [reply]
Thanks! that's solved the ping issue. I'd love to share my helm chart once I'll get this working.
